frontend/src/hooks/useKitspaceYAML.js

```diff
@@ -1,7 +1,7 @@
 import useSWR from 'swr'
 import getConfig from 'next/config'
 
-const fetcher = (...args) => fetch(...args).then(r => r.json())
+import { fetcher } from './_s3HeadFetcher'
```

Comment:
What's with the leading underscore?

Reply:
I want to keep the file in the `hooks` dir, but it isn't a hook. So I am trying to signal this.

Reply:
Let's not start doing that. Feels like you are working around the issue that having a "hooks" folder doesn't make sense. I'm sure it's a matter of taste but I'd rather the code base was organized around the features of our app and less around React or Next internals. Once we reach alpha-2 it would probably be a good time to re-organize it.

Reply:
Removed the dash.